### PR TITLE
Ve'katak Phalanx Uniforms

### PR DESCRIPTION
**Pull request by RustingWithYou**
Adds security and first responder uniforms for the Ve'katak Phalanx at the request of Vaurca lore.
Security: 
image
image

Medical:
image
image

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!review

**Review by alsoandanswer — CHANGES_REQUESTED** [review]
I need you to change the colors for the Ve'katak uniforms. They look identical to Zavodskoi.

**Comment by kyres1** [comment]
Remove the camouflaged uniform please. As explained on discord, there is no reason for a corporation to possess camouflaged uniformed security in all technicality. It's too adjacent to military use cases that ultimately security does not fall under nor represent.

**Review by kyres1 — CHANGES_REQUESTED** [review]
Remove the camouflaged uniform please. As explained on discord, there is no reason for a corporation to possess camouflaged uniformed security in all technicality. It's too adjacent to military use cases that ultimately security does not fall under nor represent.

File: code/modules/clothing/factions/pmcg.dm [hunk 1]
```diff
@@ -64,13 +64,23 @@
...
+	name = "\improper Ve'katak Phalanx uniform"
+	desc = "A security uniforn worn by members of the Ve'katak Phalanx, a Vaurca-run PMCG subsidiary. It is simple and utilitarian in its design, with the only embellishment being two red stripes on the shoulders."
+	icon_state = "phalanx_fatigues"
+	item_state = "phalanx_fatigues"
+
+/obj/item/clothing/under/rank/medical/first_responder/pmc/vekatak_phalanx
+	icon_state = "phalanx_medical"
+	item_state = "phalanx_medical"
+	name = "\improper Ve'katak Phalanx uniform"
+	desc = "A uniform worn by medical staff of the Ve'katak Phalanx, a Vaurca-run PMCG subsidiary. It is simple and utilitarian in its design, with the only embellishment being two red stripes on the shoulders."
```
Comment (SleepyGemmy) [inline code/modules/clothing/factions/pmcg.dm:86]:
```suggestion
	name = "\improper Ve'katak Phalanx uniform"
	desc = "A uniform worn by medical staff of the Ve'katak Phalanx, a Vaurca-run PMCG subsidiary. It is simple and utilitarian in its design, with the only embellishment being two red stripes on the shoulders."
	icon_state = "phalanx_medical"
	item_state = "phalanx_medical"
```

**Comment by RustingWithYou** [comment]
> I need you to change the colors for the Ve'katak uniforms. They look identical to Zavodskoi.

The sprites have been changed to be more distinct from other corporations, as well as being identifiably PMCG.

**Review by alsoandanswer — APPROVED** [review]

File: code/modules/client/preference_setup/loadout/items/factions.dm [hunk 2]
```diff
@@ -436,8 +437,11 @@
 	epmc_uniform_fr_med["EPMC first responder uniform"] = /obj/item/clothing/under/rank/medical/first_responder/pmc/epmc
 	epmc_uniform_fr_med["PMCG first responder uniform, alt"] = /obj/item/clothing/under/rank/medical/first_responder/pmc/alt
 	epmc_uniform_fr_med["Sekhmet Intergalactic first responder uniform"] = /obj/item/clothing/under/rank/medical/first_responder/pmc/sekh
+	epmc_uniform_fr_med["Ve'katak Phalanx first responder uniform"] = /obj/item/clothing/under/rank/medical/first_responder/pmc/vekatak_phalanx
 	gear_tweaks += new /datum/gear_tweak/path(epmc_uniform_fr_med)
 
+
+
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/items/factions.dm:444]:
```suggestion
```

**Review by SleepyGemmy — APPROVED** [review]

**Review by alsoandanswer — CHANGES_REQUESTED** [review]
medical is missing the PMCG highlights as on the sec uniform as according to icondiffbot

**Comment by RustingWithYou** [comment]
> medical is missing the PMCG highlights as on the sec uniform as according to icondiffbot

whoops, forgot to add them back when doing the color switch. should be fixed now

**Comment by alsoandanswer** [comment]
Both medical and security uniforms missing leg cuffs, please fix, thanks.

**Review by alsoandanswer — APPROVED** [review]
Thank you, fine work.

**Pull request merged**